playground/public/starshot.html

```diff
@@ -111,6 +111,13 @@
                 params: {
                     flavor,
                     artifact,
+                    installParameters: {
+                        skip: true,
```

Comment:
Starshot breaks because there is no cURL so the download of oEmbed providers is broken